athena-hbase/src/main/java/com/amazonaws/athena/connectors/hbase/HbaseMetadataHandler.java

```diff
@@ -278,4 +306,37 @@ protected Field convertField(String name, String glueType)
     {
         return GlueFieldLexer.lex(name, glueType);
     }
+
+    @Override
+    public GetDataSourceCapabilitiesResponse doGetDataSourceCapabilities(BlockAllocator allocator, GetDataSourceCapabilitiesRequest request)
```

Comment:
NIT: please move this after the constructor; it defines the data source capabilities and its usually lives there for all connectors; easier to find.

Reply:
Hi @AbdulR3hman done.